aireview/poetry.lock

```diff
@@ -1,5 +1,188 @@
 # This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
 
```

Comment:
**documentation**: Changes to poetry.lock should be committed to version control to ensure dependency version consistency across all environments. Ensure the changes were generated through proper 'poetry add' or 'poetry update' commands.